### ISSUE TITLE
KG APIs not to return info about projects user has no access to

### ISSUE THREAD

**Issue by jachro** [SwissDataScienceCenter/renku-graph#261]
As a Renku user, I don't want metadata of my or my organisation private/internal projects to be returned by the KG API to users whom I or my organisation didn't give access to.

Acceptance criteria:
* makes sure the knowledge-graph endpoints does not return data to users who were not authorised for (perhaps this issue should be blocked by an issue introducing security on the KG endpoints?)

**Comment by jachro** [comment]
A duplicate of #436
